frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.test.ts

```diff
@@ -669,7 +669,7 @@ describe('filtersToQueryNode', () => {
                 returning_entity: { id: '1' },
                 target_entity: { id: '1' },
                 period: RetentionPeriod.Day,
-                show_mean: true,
+                mean_retention_calculation: 'simple',
```

Comment:
nit: would be better to test that a `show_mean` filter gets converted to a `meanRetentionCalculation` field on the query. legacy queries exist with legacy properties.